packages/gasket-plugin-middleware/lib/utils.js

```diff
@@ -52,30 +44,31 @@ function applyMiddlewareConfig(middleware, plugin, middlewareConfig, middlewareP
   const configEntry = middlewareConfig.find(mw => mw.plugin === pluginName);
 
   if (configEntry) {
-    middleware.paths = configEntry.paths || [];
+    // @ts-ignore
+    middleware.paths = configEntry.paths;
     if (middlewarePattern) {
+      // @ts-ignore
       middleware.paths.push(middlewarePattern);
     }
   }
 }
 
 /**
  * Attaches the middleware layers to the app.
- * @type {import('./internal').applyMiddlewaresToApp}
+ *  @type {import('./internal').applyMiddlewaresToApp}
  */
 function applyMiddlewaresToApp(app, middlewares, middlewarePattern) {
-  if (!canUseMiddleware(app)) {
-    debug('Skipping middleware application – not an Express app');
```

Comment:
Were these checks to protect Fastify apps? It's not clear to me if making these checks will not cause problems.

Reply:
I didn't make the changes and I'm not familiar. I was out on leave when they occurred. @kbader-godaddy made the changes.